client/components/paginate/MainMenu.tsx

```diff
@@ -9,22 +8,22 @@ const MainMenu = () => {
             <h2 className="main-menu-core-functions-heading">Core Functions</h2>
             <ul className="main-menu-core-functions-list">
                 <li>
-                    <Link to="/paginate">Job Paginator</Link>
+                    <Link href="/paginate">Job Paginator</Link>
                 </li>
                 <li>
-                    <Link to="/edit">Object Editor</Link>
+                    <Link href="/edit">Object Editor</Link>
                 </li>
             </ul>
             <h2 className="main-menu-other-tools-heading">Other Tools</h2>
             <ul className="main-menu-other-tools-list">
                 <li>
-                    <a href={baseUrl + "/queue"}>Arena (Job Queue Manager)</a>
+                    <a href={"/queue"}>Arena (Job Queue Manager)</a>
```

Comment:
I think this introduces a bug: Arena is mounted as part of the back-end Express app, not the front-end React app. Thus, the link needs to point at the back-end URL.

Reply:
Made change.

Reply:
Made change.